terraspace/config/terraform/provider.tf

```diff
@@ -0,0 +1,12 @@
+provider "aws" {
+  default_tags {
+    tags = {
+      Team           = "NearForm"
+      Project        = "IPFS-Elastic-Provider"
+      Repository     = "https://github.com/web3-storage/ipfs-elastic-provider-infrastructure"
+      Environment    = "<%= expansion(':ENV') %>"
```

Comment:
FYI another way to represent this is `<%= Terraspace.env %>

Reply:
I know, I choose to use it like this for consistency with the other variables. 
It is way easier to find docs about "expansion" than "Terraspace."